src/mosaic/similarity.py

```diff
@@ -107,7 +107,7 @@ class Similarity(BaseEstimator):
     $$\rho_{X,Y} =
     \frac{\langle(X -\mu_X)(Y -\mu_Y)\rangle}{\sigma_X\sigma_Y}$$
     where for the online (low memory) algorithm the Welford algorithm taken
-    from Donald E. Knuth were used [^2].
+    from Donald E. Knuth is used [^2].
```

Comment:
Shouldn't it be:

using the Welford algorithm from Donald E. Knuth[^2] for the online (low memory) algorithm.